plugins/activity-resources/src/components/activity-message/ActivityMessageTemplate.svelte

```diff
@@ -47,17 +50,17 @@
 
   const client = getClient()
   let allActionIds: string[] = []
-  const notifyActions = [
-    notification.action.MarkAsUnreadInboxNotification,
-    notification.action.MarkAsReadInboxNotification,
-    notification.action.DeleteInboxNotification
+  const notifyActions: Action[] = [
+    // notification.action.MarkAsUnreadInboxNotification,
```

Comment:
Should we uncomment this?

Reply:
Thanks, I forgot about that
Removed excluding hereand adjusted `visibilityTester`